Kelvin_Lau_Project1.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Kelvin_Lau_Project1</title>
+    <link href="../CSS/style.css" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <img id="aloha" src="../Assets/images/aloha-logo.svg" >
+      <nav>
+        <ul>
+          <li>ABOUT</li>
+          <li>SHOP</li>
+          <li>FEATURED</li>
+          <li>UPDATES</li>
+        </ul>
+      </nav>
+      <img id="cart" src="../Assets/images/cart-icon.svg" >
+    </header>
+    <div class= "container">
+      <div class="herobanner">
+        <h6>NEW COLLECTION</h6>
+        <h2>Spring Summer 2016</h2>
+      </div>    
+    </div>  
+    <div class="container">
+      <div class="handicap">
+       <h3>Hand-Crafted Styles</h3>
+        <p>Aloha Apparel sells only the finest ready-to-travel clothing. Browse our  <br> current selection products or sign-up for updates below. </p> 
+      </div>
+    </div>
+      <div class="category">
+        <ul>
+          <li> <a href="#">
+            MEN'S </a></li>
+          <li><a href="#">
+            TRENDS</a></li>
+          <li><a href="#">
+            GLASSES</a></li>
+          <li><a href="#">
+            SALE</a></li>
+          <li><a href="#">
+            BAGS</a></li>
+          <li><a href="#">
+            WOMEN'S</a></li>
+        </ul>
+      </div>
+      <div class="selection">
```

Comment:
Use a more semantic tags instead of div when possible. For example, try using a section tag instead of div with a class of section.